sdk/communication/Azure.Communication.Common/src/AutoRefreshTokenCredential.cs

```diff
@@ -39,6 +40,7 @@ public AutoRefreshTokenCredential(CommunicationTokenRefreshOptions options)
                     RefreshAsync,
                     options.RefreshProactively,
                     initialValue: JwtTokenParser.CreateAccessToken(options.InitialToken),
+                    options.RefreshOffsetTime,
```

Comment:
@DominikMe What do you think of the name of the property `RefreshOffsetTime`? We were having a [discussion](https://microsoft-my.sharepoint.com/:w:/r/personal/martinbarnas_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B25A2A677-C34B-4EFB-BF81-1978C113C37E%7D&file=AutoRefreshToken.docx&nav=eyJjIjoyMDk3MDAwMTI4fQ&action=default&mobileredirect=true&share=IQF3pqIlS8P7Tr-BGXjBE8N-AT8CCM9qSHMOboliXSWMAxE) with the team and this is the outcome. I don't think it's bad but I'm still not 100% convinced that it's the best name. Do you think it will pass the API review?

Reply:
I think we need a very good comment explaining this regardless of the name.

It's not a bad name, but also not quite clear on first reading. Maybe `RefreshTimeBeforeTokenExpiry`?

Reply:
quite long but more descriptive. let's go with this for now. thanks @DominikMe !